app/models/concerns/requests/illiad.rb

```diff
@@ -104,5 +123,10 @@ def get_identifier(type, sub_scheme, referent, options = {})
       end
     end
     ### end code from umlaut
+
+    def ill_api_key
```

Comment:
See the SCSB mixin. Check for the Rails environment and return a "TESTME" string if the Test environment has been invoked.